automated-comments/ruby/two-fer/string_building.md

```diff
@@ -1 +1,9 @@
-In Ruby, it is unusual to use string addition. People tend to use `sprintf`, `format`, `%%`, or string interpolation. So instead of `"One for " + %<name_variable>s + ", one for me."`, consider `"One for #{%<name_variable>s}, one for me."`.
+String concatenation is perfectly valid within Ruby, but a more common way to compose a string is with [string interpolation](http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html). For example:
```

Comment:
A small note: this comment is named `string_building`, but aren't the other options also about string building? Maybe `string_concatenation` would be a bit more precise?

Reply:
Nice. Yes. Done, thanks.